src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -90,6 +92,15 @@ public async Task ConfigRevisions()
                         }
                     }
                 });
+
+            if (enforce)
+                await EnforceConfigRevisions();
+        }
+
+        [RavenAction("/databases/*/admin/revisions/config/enforce", "POST", AuthorizationStatus.DatabaseAdmin)]
+        public async Task EnforceConfigRevisions()
+        {
+            await Database.DocumentsStorage.RevisionsStorage.EnforceConfiguration();
```

Comment:
This can take a long time. We should wrap this in an operation, return OperationId and create new `EnforceRevisionsConfigurationOperation` in Client API